<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QWidget" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text">
          <string>下载线程数</string>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QComboBox" name="encodeSelect"/>
       </item>
       <item row="5" column="2">
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>是否启用</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>解码器（需重启）</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>看图时模型</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="checkBox_2">
         <property name="text">
          <string>启用代理</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>画质选择</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>下载时模型</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>线程数</string>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QComboBox" name="lookModel">
         <item>
          <property name="text">
           <string>自动</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cunet（效果最好）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>photo（写真）（速度快）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>anime_style_art_rgb（动漫）（速度快）</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QComboBox" name="threadSelect">
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLineEdit" name="saveEdit"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="10" column="2">
        <widget class="QComboBox" name="downloadModel">
         <item>
          <property name="text">
           <string>自动</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>cunet（效果好）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>photo（写真）（速度快）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>anime_style_art_rgb（动漫）（速度快）</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>waifu2x设置</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="quality_original">
           <property name="text">
            <string>原画</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="quality_high">
           <property name="text">
            <string>高</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="quality_medium">
           <property name="text">
            <string>中</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="quality_low">
           <property name="text">
            <string>低</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="httpEdit">
         <property name="placeholderText">
          <string>http://127.0.0.1:10809</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>聊天室</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="comboBox">
         <property name="placeholderText">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>http代理</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>保存路径</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QComboBox" name="logBox">
         <item>
          <property name="text">
           <string>WARN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>INFO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DEBUG</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>日志等级</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Setting</receiver>
   <slot>SaveSetting()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>336</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Setting</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>137</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Setting</receiver>
   <slot>SelectSavePath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SaveSetting()</slot>
  <slot>SelectSavePath()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
